src/common/types/cdn.types.ts

```diff
@@ -39,6 +41,8 @@ export interface SuperVizCdn {
   MeetingConnectionStatus: typeof MeetingConnectionStatus;
   MeetingControlsEvent: typeof MeetingControlsEvent;
   ParticipantEvent: typeof ParticipantEvent;
+  WhoIsOnlineEvent: typeof WhoIsOnlineEvent;
+  ComponentLifeCycleEvent: typeof ComponentLifeCycleEvent;
```

Comment:
```suggestion
  WhoIsOnlineEvent: typeof WhoIsOnlineEvent;
  ComponentLifeCycleEvent: typeof ComponentLifeCycleEvent;
  CommentEvent: typeof CommentEvent
```

Reply:
`CommentEvent` is a few alines above `ComponentLifeCycleEvent` in the original code, in line 36 of the file